init.lua

```diff
@@ -669,26 +669,45 @@ local function bisGUI()
 			ImGui.PushStyleVar(ImGuiStyleVar.ScrollbarSize, 17)
 			if ImGui.BeginTabBar('bistabs') then
 				if ImGui.BeginTabItem('Gear') then
+					if settings['SelectedList'] then
```

Comment:
this loop would run every frame it seems like? it would only make sense to run this one time at startup. i would move this into the spot where i currently am setting the list at startup based on what zone you are in.
something like "if we're in a zone the corresponds to one of the lists, pick that list, otherwise show the last selected list"